### ISSUE TITLE
Suggested new recipe discussion.

### ISSUE THREAD

**Issue by milkypostman** [melpa/melpa#4385]
/ping @purcell @tarsius @syohex 

What would you think about adding some sort of "sponsorship" program where in order to actually have a package reviewed there needs to be one other person who will 👍 the PR? I hate to call it a "vote" but given the sheer number of requests lately it seems like it would be nice to have a community sponsorship deal where someone else has to basically say "hey I use this package too."

This would avoid things where someone happens to have written a clever package but where it's not really relevant or interesting to anyone else.

Just a thought.

**Comment by purcell** [comment]
Yep, I can see that working, and I guess we could still 👍 the PRs ourselves if we thought it looked worthwhile. My concern would be that the process would be less welcoming to contributors, and we might not have any visibility of cases wherein this policy deters people from submitting great packages. Probably overall I'm a bit "meh" on this specific proposal -- we could just as easily note explicitly that packages may be rejected freely if they do not appear to be ready for distribution or of widespread usefulness.

**Comment by Fanael** [comment]
The obvious concern is that I'd presume that most Emacs users never visit the MELPA repo on GitHub, so it's entirely possible for a package to have some users but never receive a :+1:.

> we could just as easily note explicitly that packages may be rejected freely if they do not appear to be ready for distribution or of widespread usefulness

This is the current policy anyway, isn't it?

**Comment by purcell** [comment]
> This is the current policy anyway, isn't it?

Yes, but it's not written anywhere.

**Comment by tarsius** [comment]
I think the issue is that kids are spoiled these days. Seriously though, I get the impression that authors add their packages to Melpa as early as possible because they find it more convenient to install it that way than to manually adjust the `load-path` to point to the development repository.

So maybe we should tell authors of very young packages to first polish it over the course of at least one month before submitting to Melpa. On the other hand many authors seem to need the review the receive here.

I guess I vote for business as usual, i.e. no new sponsorship requirement.
